app/src/main/res/values/colors.xml

```diff
@@ -5,6 +5,7 @@
 
     <color name="colorPrimary">#008577</color>
     <color name="colorPrimaryDark">#00574B</color>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/ar-localizer-view-android/blob/6e7a43f049fda9487b1419ac862329af39e581fa/app/src/main/res/values/colors.xml#L7"></span>The resource <code>R.color.colorPrimaryDark</code> appears to be unused</td>
    </tr>
  </tbody>
</table>